docs/components/accordion/accordion.njk

```diff
@@ -97,12 +97,6 @@ An Accordion is a vertically stacked set of headings that each control the visib
             <button class="jazz-accordion__button" aria-expanded="true" aria-controls="content2">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Cras ultricies ligula sed magna dictum porta?</button>
           {% endhighlight %}
     </li>
-    <li>The <span class="code">hidden</span> attribute of an accordion panel should not be set explicitly to allow the content to be usable even if browser scripting is not enabled or fails to execute.
```

Comment:
Please add a short blurb about adding the jazz-accordion__content--expanded class to control visibility of the content.